examples/azure/lb-example/README.md

```diff
@@ -0,0 +1,21 @@
+# AKS Load Balancer type deployment
+
+This folder contains examples of using GraphDB with the Azure Load Balancer and Application Gateway Ingress.
```

Comment:
Wait, this example is for a Service of type LoadBalancer, why do we need the Ingress here?

Reply:
Because of the TLS which is not supported in the Azure Load Balancer since it works on Layer4, but after a discussion will be removed.

Reply:
Update the readme to reflect the values.yaml